Assets/MXR.SDK/Runtime/Android/Utils/MXRAndroidUtils.cs

```diff
@@ -107,6 +109,27 @@ public static void SendBroadcastAction(string action) {
                 Debug.unityLogger.Log(LogType.Error, "Could not broadcast action " + action);
         }
 
+        /// <summary>
+        /// Opens the browser from a resume action if the admin app version is sufficient. Otherwise, 
+        /// opens the browser via Unity's Application.OpenURL. We pass the url through an intent to the
+        /// android app to open the browser if the admin app fails as well.
+        /// </summary>
+        public static void OpenBrowserFromResume(String url)
+        {        
```

Comment:
nit: @tristanmanage could you move the brace same line. I've made a PR to add a `.editorconfig` file so it should take care of this in the future

Reply:
yeah ofc changed!